samples/oneapi/dpc/mpi/sources/basic_statistics_distr_mpi.cpp

```diff
@@ -17,68 +17,68 @@
 #include <CL/sycl.hpp>
 #include <iomanip>
 #include <iostream>
-#include <mpi.h>
 
 #ifndef ONEDAL_DATA_PARALLEL
 #define ONEDAL_DATA_PARALLEL
 #endif
 
 #include "oneapi/dal/algo/basic_statistics.hpp"
-#include "oneapi/dal/spmd/mpi/communicator.hpp"
 #include "oneapi/dal/io/csv.hpp"
+#include "oneapi/dal/spmd/mpi/communicator.hpp"
 
 #include "utils.hpp"
 
 namespace dal = oneapi::dal;
 
-void run(sycl::queue& queue) {
-  const auto data_file_name = get_data_path("data/covcormoments_dense.csv");
+void run(sycl::queue &queue) {
+    const auto data_file_name = get_data_path("data/covcormoments_dense.csv");
 
-  const auto data = dal::read<dal::table>(
-      queue, dal::csv::data_source{data_file_name});
+    const auto data =
+        dal::read<dal::table>(queue, dal::csv::data_source{data_file_name});
 
-  const auto bs_desc = dal::basic_statistics::descriptor{};
+    const auto bs_desc = dal::basic_statistics::descriptor{};
 
-  auto comm = dal::preview::spmd::make_communicator<dal::preview::spmd::backend::mpi>(queue);
-  auto rank_id = comm.get_rank();
-  auto rank_count = comm.get_rank_count();
+    auto comm =
+        dal::preview::spmd::make_communicator<dal::preview::spmd::backend::mpi>(
+            queue);
+    auto rank_id = comm.get_rank();
+    auto rank_count = comm.get_rank_count();
 
-  auto input_vec =
-      split_table_by_rows<float>(queue, data, rank_count);
+    auto input_vec = split_table_by_rows<float>(queue, data, rank_count);
 
-  const auto result = dal::preview::compute(comm, bs_desc, input_vec[rank_id]);
-  if (comm.get_rank() == 0) {
-    std::cout << "Minimum:\n" << result.get_min() << std::endl;
-    std::cout << "Maximum:\n" << result.get_max() << std::endl;
-    std::cout << "Sum:\n" << result.get_sum() << std::endl;
-    std::cout << "Sum of squares:\n" << result.get_sum_squares() << std::endl;
-    std::cout << "Sum of squared difference from the means:\n"
-              << result.get_sum_squares_centered() << std::endl;
-    std::cout << "Mean:\n" << result.get_mean() << std::endl;
-    std::cout << "Second order raw moment:\n"
-              << result.get_second_order_raw_moment() << std::endl;
-    std::cout << "Variance:\n" << result.get_variance() << std::endl;
-    std::cout << "Standard deviation:\n"
-              << result.get_standard_deviation() << std::endl;
-    std::cout << "Variation:\n" << result.get_variation() << std::endl;
-  }
+    const auto result = dal::preview::compute(comm, bs_desc, input_vec[rank_id]);
+    if (comm.get_rank() == 0) {
```

Comment:
```suggestion
     if (rank_id == 0) {
```

Reply:
changed all conditions to if (comm.get_rank() == 0) { to get it in one style